src/uint/boxed.rs

```diff
@@ -395,17 +398,75 @@ impl fmt::UpperHex for BoxedUint {
     }
 }
 
+#[cfg(feature = "serde")]
+impl<'de> Deserialize<'de> for BoxedUint
```

Comment:
Encoding-related functionality should go under https://github.com/RustCrypto/crypto-bigint/blob/master/src/uint/boxed/encoding.rs

Reply:
Thanks done